tests: drivers: build_all: build failure for NXP dac_mcux_gau.c

### Describe the bug

Platform:
- frdm_rw612
- rd_rw612 

c73e59c5f4ca0742512a8278e337b6762f51bc38 is the first bad commit
commit c73e59c5f4ca0742512a8278e337b6762f51bc38
Author: Tomas Groth Christensen <tgc@foss.dk>
Date:   Tue Jan 27 13:29:22 2026 +0100

    dac: mcux_gau: fix output-voltage-range mapping on RW61x
    
    Map DT range index to SDK values {0,1,3} on RW61x
    
    Signed-off-by: Tomas Groth Christensen <tgc@foss.dk>

 drivers/dac/dac_mcux_gau.c | 24 ++++++++++++++----------
 1 file changed, 14 insertions(+), 10 deletions(-)

### Regression

- [x] This is a regression.

### Steps to reproduce

west build -b frdm_rw612 tests/drivers/build_all

### Relevant log output

```shell
/home/jenkins/agent/workspace/zephyr_upstream_build_all_driver/zephyr/include/zephyr/device.h:
261:9: note: in expansion of macro 'DEVICE_DT_DEINIT_DEFINE' 261 | DEVICE_DT_DEINIT_DEFINE(node_id, init_fn, NULL, pm, data, config, \
 | ^~~~~~~~~~~~~~~~~~~~~~~/home/jenkins/agent/workspace/zephyr_upstream_build_all_driver/zephyr/include/zephyr/device.h:
284:9: note: in expansion of macro 'DEVICE_DT_DEFINE' 284 | DEVICE_DT_DEFINE(DT_DRV_INST(inst), __VA_ARGS__)
 | ^~~~~~~~~~~~~~~~/home/jenkins/agent/workspace/zephyr_upstream_build_all_driver/zephyr/drivers/dac/dac_mcux_gau.c:119:9: note: in expansion of macro 'DEVICE_DT_INST_DEFINE' 119 
| DEVICE_DT_INST_DEFINE(inst, &nxp_gau_dac_init, NULL, \
 | ^~~~~~~~~~~~~~~~~~~~~/home/jenkins/agent/workspace/zephyr_upstream_build_all_driver/zephyr/twister-out/frdm_rw612_rw612/zephyr/tests/drivers/build_all/led/drivers.led.build/zephyr/include/generated/zephyr/devicetree_generated.h:
23019:46: note: in expansion of macro 'NXP_GAU_DAC_INIT' 23019 
| #define DT_FOREACH_OKAY_INST_nxp_gau_dac(fn) fn(0) | ^~/home/jenkins/agent/workspace/zephyr_upstream_build_all_driver/zephyr/include/zephyr/sys/util_internal.h:
146:36: note: in expansion of macro 'DT_FOREACH_OKAY_INST_nxp_gau_dac' 
146 | #define UTIL_PRIMITIVE_CAT(a, ...) a##__VA_ARGS__
       | ^/home/jenkins/agent/workspace/zephyr_upstream_build_all_driver/zephyr/drivers/dac/dac_mcux_gau.c:
125:1: note: in expansion of macro 'DT_INST_FOREACH_STATUS_OKAY' 
125 | DT_INST_FOREACH_STATUS_OKAY(NXP_GAU_DAC_INIT)
       | ^~~~~~~~~~~~~~~~~~~~~~~~~~~ninja: build stopped: subcommand failed.
```

### Impact

Functional Limitation – Some features not working as expected, but system usable.

### Environment

- os: ubuntu
- toolchain: zephyr
- version: v4.3.0-5056-gcf5deb7c6e2e

### Additional Context

_No response_

--- a/drivers/dac/dac_mcux_gau.c
+++ b/drivers/dac/dac_mcux_gau.c
@@ -101,7 +101,7 @@ static int nxp_gau_dac_init(const struct device *dev)
 	return 0;
 };
 
-/* Map output range enum index to mcux dac_output_voltage_range_t */ /
+/* Map output range enum index to mcux dac_output_voltage_range_t */
 #define _NXP_MAP_OUTPUT_RANGE(idx)	\
 	((idx) == 0 ? kDAC_RangeSmall :	\
 	(idx) == 1 ? kDAC_RangeMiddle :	\

